scripts/verify_sitemap_links.py

```
@@ -0,0 +1,123 @@
+import sys
+import xml.etree.ElementTree as ET
+import urllib.request
+import urllib.error
+
+def check_url(url: str) -> bool:
+    """Sends a request to verify the URL exists and is not broken."""
+    req = urllib.request.Request(
+        url,
+        method="GET",
+        headers={"User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64)"}
+    )
```

Ruff (0.16.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)